src/theme/button.ts

```diff
@@ -0,0 +1,137 @@
+import { defineStyle, defineStyleConfig } from '@chakra-ui/react';
+
+const green = defineStyle({
+  bgGradient: 'linear(87.93deg, #3AFFA7 2.84%, #55E8E2 97.16%)',
+  textColor: 'brand.darkGreen',
+  borderWidth: '2px',
+  borderRadius: 'full',
+  _hover: {
+    bg: 'brand.darkGreen',
+    borderColor: 'brand.green',
+    borderWidth: '2px',
+    textColor: 'brand.green',
+  },
+});
+
+const ghostGreen = defineStyle({
+  textColor: 'brand.green',
+  fillColor: 'brand.green',
+  bgColor: 'brand.darkGreen',
+});
+
+const white = defineStyle({
+  bg: '#F0F0F0EE',
+  borderColor: '#F0F0F0EE',
+  borderWidth: '2px',
+  textColor: 'brand.darkGreen',
+  borderRadius: 'full',
+  _hover: {
+    bg: 'brand.darkGreen',
+    borderColor: '#F0F0F0EE',
+    borderWidth: '2px',
+    textColor: '#F0F0F0EE',
+  },
+});
+
+const ghostWhite = defineStyle({
+  borderRadius: 'full',
+  color: '#F0F0F0EE',
+  cursor: 'pointer',
+  _hover: {
+    color: 'brand.green',
+  },
+});
+
+const whiteModal = defineStyle({
+  bg: '#F0F0F0EE',
+  borderColor: '#F0F0F0EE',
+  borderWidth: '2px',
+  textColor: 'brand.modalGreen',
+  borderRadius: 'full',
+  _hover: {
+    bg: 'brand.modalGreen',
+    borderColor: '#F0F0F0EE',
+    borderWidth: '2px',
+    textColor: '#F0F0F0EE',
+  },
+});
+
+const whiteOutline = defineStyle({
+  bg: 'brand.darkGreen',
+  borderColor: '#F0F0F0EE',
+  borderWidth: '2px',
+  textColor: '#F0F0F0EE',
+  borderRadius: 'full',
+  _hover: {
+    bg: 'brand.darkGreen',
+    borderColor: '#F0F0F0EE',
+    borderWidth: '2px',
+    textColor: '#F0F0F0EE',
+  },
+});
+
+const danger = defineStyle({
+  bg: 'red.600',
+  borderColor: 'red.600',
+  borderWidth: '2px',
+  textColor: 'brand.lightGray',
+  borderRadius: 'full',
+  _hover: {
+    bg: 'red.500',
+    borderColor: 'red.500',
+    borderWidth: '2px',
+    textColor: 'brand.lightGray',
+  },
+});
+
+const dark = defineStyle({
+  textColor: 'brand.lightGray',
+  fillColor: 'brand.lightGray',
+  bgColor: 'brand.darkGreen',
+  borderRadius: 'full',
+  _hover: {
+    textColor: 'brand.green',
+    fillColor: 'brand.green',
+  },
+});
+
+const outline = defineStyle({
+  bg: 'brand.darkGreen',
+  borderColor: 'brand.green',
+  borderWidth: '2px',
+  textColor: 'brand.green',
+  borderRadius: 'full',
+  _hover: {
+    bg: 'brand.darkGreen',
+    borderColor: 'brand.green',
+    borderWidth: '2px',
+    textColor: 'brand.green',
+  },
+});
+
+const ghost = defineStyle({
+  borderRadius: 'full',
+});
+
+const ghostGray = defineStyle({
+  borderRadius: 'full',
+  color: '#7D7B7B',
+});
+
+const buttonTheme = defineStyleConfig({
+  defaultProps: {
+    colorScheme: 'spacemesh',
+  },
+  variants: {
+    green,
+    ghostGreen,
+    white,
+    ghostWhite,
+    whiteModal,
+    whiteOutline,
+    danger,
+    outline,
+  },
```

Comment:
I'm not sure it is a good solution.

Chakra's theme system supposed that there is variants.
For buttons it is `solid`, `outline`, `link` and `ghost`.
And then if you want to have different styles — use `colorCheme`. It provides a variety of different colors, but for the branding we need less, so in this case it might be good to name them like `primary`, `secondary`, `danger`, and mb `primaryDark` for cases when buttons on dark bg should be different...

But here we have a mix of these, and names are not very clear when which to use. And as I said above, I don't like that all the buttons have the same accents, for example "Add party" and "Add account" looks the same, and which is even more confusing "Cancel" and "Next" the same as well :)

To sum up, I'd like to have:
1. clear naming,
2. probably separate variants and colors (if it make sense to avoid having a big amount of variants),
3. default styles for every component, so we don't need to put `variant="whiteModal"` in every component